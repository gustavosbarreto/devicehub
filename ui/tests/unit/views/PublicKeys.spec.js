import Vuex from 'vuex';
import { shallowMount, createLocalVue } from '@vue/test-utils';
import Publickeys from '@/views/PublicKeys';

describe('Publickeys', () => {
  const localVue = createLocalVue();
  localVue.use(Vuex);

  let wrapper;

  beforeEach(() => {
    wrapper = shallowMount(Publickeys, {
      localVue,
      stubs: ['fragment'],
    });
  });

  it('Is a Vue instance', () => {
    expect(wrapper).toBeTruthy();
  });
  it('Renders the component', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
