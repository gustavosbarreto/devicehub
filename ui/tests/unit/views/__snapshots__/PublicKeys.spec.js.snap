// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Publickeys Renders the component 1`] = `
<div>
  <publickey-stub data-test="publickey-component"></publickey-stub>
</div>
`;
